src/Control/Email/EmailSender.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace SilverStripe\Control\Email;
+
+use SilverStripe\Core\Injector\Injector;
+use Symfony\Component\Mailer\Messenger\SendEmailMessage;
+use Symfony\Component\Mailer\Transport\TransportInterface;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Transport\Sender\SenderInterface;
+use Symfony\Component\Mime\RawMessage;
+
+class EmailSender implements SenderInterface
+{
+    public function send(Envelope $envelope): Envelope
+    {
+        /** @var SendEmailMessage $sendEmailMessage */
+        $sendEmailMessage = $envelope->getMessage();
+        /** @var RawMessage $rawMessage */
+        $rawMessage = $sendEmailMessage->getMessage();
+        /** @var TransportInterface $transport */
+        $transport = Injector::inst()->get(TransportInterface::class);
+        $transport->send($rawMessage);
+        return $envelope;
```

Comment:
What does this do that wouldn't be achieved if we just replaced mention of `Symfony\Component\Messenger\Transport\Sender\SenderInterface.mailer` with `Symfony\Component\Mailer\Transport\TransportInterface` instead?
`TransportInterface` extends `SenderInterface` after all.

Reply:
Have deleted file, no longer needed now the bus is gone